spring/spring6/spring6-common/src/main/java/org/infinispan/spring/common/provider/SpringCache.java

```diff
@@ -32,6 +36,7 @@ public class SpringCache implements Cache {
    private final long readTimeout;
    private final long writeTimeout;
    private final Map<Object, ReentrantLock> synchronousGetLocks = new ConcurrentHashMap<>();
```

Comment:
Technically the synchronous version can use the other map, but I am fine leaving it how it is.